src/ui/entry.ts

```diff
@@ -195,6 +228,98 @@ async function getCachedPassphrase() {
       });
 }
 
+function getExportDataFromOTPAuthPerLine(importCode: string) {
```

Comment:
Importing codes with certian characters in the account name field don't import the name successfully.
```
otpauth://totp/%20:%3A%3A%3A!!!?secret=aaaabbbbcccc&issuer=%20&period=30
```

Reply:
I believe this is a broken data. The label of otpauth URL should be fully encoded, so `!` should never be there.

Reply:
Must be a problem with encoding. I was able to make this without manually editing the backup file
```
otpauth://totp/?secret=QP2RFPQGO4TKEQ64K5KDYJ7KU2NNEHVO
otpauth://totp/✓ à la mode:example.com?secret=PKX5BHAPN63RQE726HWXW2VPA72AMPOG&issuer=✓ à la mode
otpauth://totp/!!!"""'''&&==++##::?secret=AFKVXHTAZZQKCHI3XSZPX5NKQRFXL3AD&issuer=!!!"""'''&&==++##:
```
![image](https://user-images.githubusercontent.com/27789806/53128887-915a7080-352b-11e9-8474-6f0438fee696.png)

%3A and : are not allowed in label section.

Reply:
How about just remove all colons when export as otpauth urls?

Reply:
That works. It seems to have stopped URI encoding issuer & account fields for some reason too.

Reply:
Fixed